README.md

```diff
@@ -55,6 +55,8 @@ Second, the plugin looks for the `CASC_JENKINS_CONFIG` environment variable. The
 - A full path to a single file. For example, `/var/jenkins_home/casc_configs/jenkins.yaml`.
 - A URL pointing to a file served on the web. For example, `https://acme.org/jenkins.yaml`.
 
+If `CASC_JENKINS_CONFIG` point to a folder, the plugin will recursively traversing the folder to find file(suffix with .yml,.yaml,.YAML,.YML), but dosn't contain hidden files or hidden subdirectories. It dosn't follow symbolic links.
```

Comment:
couple of typos:
"point**s**", "travers**e**", space between "file" and "(", and "do**e**sn't" and we're good to go!

Reply:
thank you.
I have modified it according to your statement.